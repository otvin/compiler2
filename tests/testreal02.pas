program testreal02;

begin
    writeln(1.2 + 2.3);
    writeln(4.19 - 0.01);
    writeln(2.1 - 4.2);
    writeln(12.0 * 0.5);
end.
