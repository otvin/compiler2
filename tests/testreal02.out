3.500000000000
4.180000000000
-2.100000000000
6.000000000000
